### PR TITLE
many, many fixes for openshift-logging and related code cleanup

### PR DESCRIPTION
**Pull request by richm**

### CODE REVIEW

**Comment by richm** [comment]
@jcantrill @ewolinetz @nhosoi I think this PR should supercede https://github.com/openshift/origin-aggregated-logging/pull/1071

File: test/access_control.sh [hunk 1]
```diff
@@ -260,11 +261,18 @@ oc project ${LOGGING_NS} > /dev/null
 test_user_has_proper_access $LOG_NORMAL_USER $LOG_NORMAL_PW access-control-1 access-control-2 -- access-control-3
 test_user_has_proper_access $LOG_USER2 $LOG_PW2 access-control-2 access-control-3 -- access-control-1
 
+logging_index=".operations.*"
+if [ ${LOGGING_NS} = "logging" ] ; then
```
Comment (ewolinetz) [inline test/access_control.sh]: what about if ${LOGGING_NS} = "openshift-logging" ?

File: test/access_control.sh [hunk 2]
```diff
@@ -260,11 +261,18 @@ oc project ${LOGGING_NS} > /dev/null
...
+  logging_index="project.logging.*"
+fi
+
 os::log::info now auth using admin + token
 get_test_user_token $LOG_ADMIN_USER $LOG_ADMIN_PW
-nrecs=$( curl_es_with_token $espod "/project.${LOGGING_NS}.*/_count" $test_name $test_token | \
-         get_count_from_json )
-os::cmd::expect_success "test $nrecs -gt 1"
+if [ ${LOGGING_NS} = "logging" ] && [ $espod != $esopspod] ; then
```
Comment (ewolinetz) [inline test/access_control.sh]: Should we also test for `openshift-logging`?

**Comment by richm** [comment]
tests finally passing - can I get a re-lgtm?

**Comment by ewolinetz** [comment]
/lgtm

On Apr 16, 2018 10:01 PM, "Richard Megginson" <notifications@github.com>
wrote:

tests finally passing - can I get a re-lgtm?

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin-aggregated-logging/pull/1083#issuecomment-381818087>,
or mute the thread

**Pull request merged**